code/style.css

```diff
@@ -1,4 +1,309 @@
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  padding: 14px;
+}
+
+@media (min-width: 768px) {
+  .big-news-card {
+    flex-wrap: nowrap;
+    width: 100%;    
+    justify-content: space-evenly;
+  }
+}
+
+@media (min-width: 1024px) {
+  .big-news-card {
+    width: 100%;
+  }
+}
+
+.big-news-card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.big-news-article a {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 14px;
+  text-align: left;
+  color: #DB3334;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  text-transform: uppercase;
+  font-size: 18px;
+  text-align: left;
+  color: #000000;
+  margin: 10px 0px 0px 0px;
+}
+
+/** SMALLER NEWS SECTION **/
+
+.small-news {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  margin: 20px 0px 0px 0px;
+}
+
+@media (min-width: 1024px) {
+  .small-news {
+    margin: 25px 0px 0px 0px;
+  }
+}
+
+/** SMALLER NEWS TEXT BOX**/
+
+a {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  text-align: left;
+  color: #DB3334;
+  margin: 0px 0px 0px 5px;
+}
+
+@media (min-width: 1024px) {
+  a {
+    font-size: 10px;
+  }
+}
+
+h2 {
+  font-family: 'Playfair Display', serif;
+  text-transform: uppercase;
+  font-size: 16px;
+  text-align: left;
+  color: #000000;
+  margin: 5px;
+  line-height: 1.2;
+}
+
+@media (min-width: 1024px) {
+  h2 {
+    font-size: 12px;
+  }
+}
+
+.small-news p {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 10px;
+  font-weight: bolder;
+  text-align: left;
+  color: #000000;
+  margin: 5px;
+}
+
+.small-news img {
+  width: 40%;
+  height: 90%;
+  align-self: center;
+}
+
+@media (min-width: 1024px) {
+  .small-news img {
+    width: 40%;
+    height: 85%;
+  }
+}
+
+.small-news article {
+  width: 50%;
+  height: 130px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.small-news-card {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  padding: 5px;
+  margin-bottom: 10px;
+}
+
+@media (min-width: 768px) {
+  .small-news-card {
+    width: 48%
+  }
+}
+
+@media (min-width: 1024px) {
+  .small-news-card {
+    width: 24%;
+  }
+}
+
+@media (min-width: 1024px) {
+  .small-news-card {
+    margin-bottom: 20px;
+  }
+}
+
+/* On mouse-hover, add a deeper shadow */
+.small-news-card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+footer {
+  background: #000000;
+  right: 0;
+  left: 0;
+  padding: 10px;
+  text-align: center;
+  margin: 10px 0px 0px 0px;
+}
+
+@media (min-width: 768px) {
+  footer {
+    bottom: 0;
+  }
+}
+
+footer p {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  color: #ffffffa4;
```

Comment:
What color is that?:D Hexadecimal values should only have 6 letters (or 3, if all 6 of them are the same)